tests/unit/src/test/scala/tests/NewFileTemplateSuite.scala

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import munit.ScalaCheckSuite
+import scala.meta.internal.metals.NewFileTemplate
+import org.scalacheck.Gen
+import org.scalacheck.Prop.forAll
+
```

Comment:
/cc @olafurpg dogfooding the new scalacheck module 👇

Reply:
Do we really need Scalacheck here though? It's supposed to be used over a set of possible values and here it is just normal testing. I have never found a usage for property based testing and maybe it would be better to keep the tests consistent?

Reply:
> It's supposed to be used over a set of possible values and here it is just normal testing

I'm using Scalacheck to test "all" possible positions of the cursor inside a (fixed) document.


> Do we really need Scalacheck here though?

It's not strictly necessary, but it just gives me a little more confidence that the computation is not right by chance but it holds for "all" possible cursor positions.

That said, if you prefer we can remove it.

Reply:
> I'm using Scalacheck to test "all" possible positions of the cursor inside a (fixed) document.

I am totally blind and have not seen it. It's a perfectly valid scenario to use it in. :+1: